apps/charterafrica/src/components/ToolCard/ToolCard.js

```diff
@@ -0,0 +1,184 @@
+import { Link } from "@commons-ui/next";
+import { Box, Button, CardContent, CardMedia, SvgIcon } from "@mui/material";
+import PropTypes from "prop-types";
+import React from "react";
+
+import GithubIcon from "@/charterafrica/assets/icons/Type=github, Size=24, Color=CurrentColor.svg";
+import LineClampedRichTypography from "@/charterafrica/components/LineClampedRichTypography";
+import Card, { StyledActionArea } from "@/charterafrica/components/StyledCard";
+
+const CardActionAreaWrapper = React.forwardRef((props, ref) => {
+  const { disableCard, ...restProps } = props;
+  return disableCard ? (
+    props.children
+  ) : (
+    <StyledActionArea ref={ref} {...restProps} />
+  );
+});
```

Comment:
@kilemensi this was necessitated by this.
<img width="1017" alt="image" src="https://github.com/CodeForAfrica/ui/assets/39160236/26217188-edf8-4c35-a8c4-3584dea6719e">
We cannot have a Link button inside a clickable card. Are you suggesting we push back with the design team on the removal of this part?

Reply:
Yes @koechkevin.

Either the card should be clickable, linking to tool page on our site or the card is not clickable and there should be a read more or another link that will link to the tool page on our site.

Without the above, all the card does is link to GitHub instead of our own page.